ha/k8s_setup/ha_start.py

```diff
@@ -30,11 +32,29 @@ def usage(prog: str):
             f"--services health_monitor, fault_tolerance, k8s_monitor\n"
             f"--config   Config URL\n")
 
+def handle_signal(signum, frame):
+    """
+    Sif the process is started already then forward the received signal
+    """
+    print(f"Signal {signum} at frame {frame} received.")
+    print(f"Sending {signum} to driver process pid: {driver_process.pid}.")
+    driver_process.send_signal(signum)
+
 try:
     if args.services not in service_entry_mapping:
         print('Please provide a valid service name')
         usage('ha_start')
         sys.exit(22)
+    # aetting Signal handlers for all standered signals
```

Comment:
Why there is a need to handle all the standard signals?

Reply:
just to get all the signals from Kubernetes, don't know which signal will be sent by  Kubernetes in which scenarios.
as our process is a child process, the not exactly main process of the container and Kubernetes signal is sent to only the main process of the container, not a child process so I want to make sure all signals are getting transfer to our process which suppose be the main process of container.

Reply:
Ok. Thanks!